documentation/docs/steps/neoDeploy.md

```diff
@@ -4,7 +4,7 @@
 
 ## Prerequisites
 
-* **SAP CP account** - the account to where the application is deployed.
+* **SAP CP account** - the account to where the application is deployed. To deploy MTA (`deployMode: MTA`) an over existing application you need a _Java Quota_ of at least 2.
```

Comment:
Is it always a java quota of 2? Maybe a java quota with at least one item left/unused fits better here. And that does only apply for  java like MTA deployments. For html5 based apps there seems to be no quota at all.

Reply:
correct, thanks a lot. I fixed that (hopefully) in the following commits.

Reply:
maybe nit-pick: for html5 there is apparently no quota at all. So this applies for java applications.